agent/exec/dockerapi/container.go

```diff
@@ -442,7 +443,7 @@ func (c *containerConfig) resources() enginecontainer.Resources {
 	// set pids limit
 	pidsLimit := c.spec().PidsLimit
 	if pidsLimit > 0 {
-		resources.PidsLimit = pidsLimit
+		resources.PidsLimit = &pidsLimit
```

Comment:
is there a reason that this changed?

Reply:
It looks to be changed on moby side and it didn't passed build without those (like I said in change log on first message)